### DISCUSSION TITLE
How to change the green primary color to another color

### DISCUSSION THREAD

**Discussion by thamibn** [tusen-ai/naive-ui#1384] [Q&A] [answered]
Hi I would like to change the green primary or common color to another color how can i change that.
I have tried the below

```

<n-config-provider :theme-overrides="themeOverrides">
//button...
</n-config-provider>


themeOverrides : {
    common: {
        primaryColor: '#FF0000'
    },
    // ...
}
```

but still the button color is green on hover.

**Comment by thamibn** [comment]
or maybe i am placing the ```n-config-provider``` on a wrong place

I have Layout.vue that has my `<main></main>` and everything get rendered there.

**Comment by thamibn** [comment]
> Hi I would like to change the green primary or common color to another color how can i change that. I have tried the below
> 
> ```
> 
> <n-config-provider :theme-overrides="themeOverrides">
> //button...
> </n-config-provider>
> 
> 
> themeOverrides : {
>     common: {
>         primaryColor: '#FF0000'
>     },
>     // ...
> }
> ```
> 
> but still the button color is green on hover.

the primaryColor is working how can i change the hover, active, or focus color from green to another color

**Reply by 07akioni** [reply]
primaryColorHover
primaryColorPressed

**Reply by 07akioni** [reply, marked as answer]
try using themeoverrides with types, that would make it easier to use

**Reply by Uhasith** [reply]
Thank you @07akioni ❤️

**Comment by Uhasith** [comment]
From where i can find these configurable keywords?  Is there any List in the Doc? For other configurations ? @07akioni 

primaryColorHover
primaryColorPressed ...... etc ???

**Reply by jahnli** [reply]
theme editor
